codec-http2/src/main/java/io/netty/handler/codec/http2/Http2ConnectionHandler.java

```diff
@@ -717,7 +717,7 @@ protected void onStreamError(ChannelHandlerContext ctx, boolean outbound,
                 try {
                     stream = encoder.connection().remote().createStream(streamId, true);
                 } catch (Http2Exception e) {
-                    resetUnknownStream(ctx, streamId, http2Ex.error().code(), ctx.newPromise());
```

Comment:
This method will be called later then via the `resetStream(...)` method that is part of the `Http2LifecycleManager` interface